pysat/instruments/template_cdaweb_instrument.py

```diff
@@ -38,11 +38,11 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
-import pandas as pds
+import functools
 import numpy as np
-import pysat
+import pandas as pds
```

Comment:
Pandas isn't a core python package, so should be after sys, but before pysat.

Reply:
fixed.